src/missing-services.ts

```diff
@@ -2290,6 +2292,24 @@ registerSingleton(IAccessibleViewService, class AccessibleViewService implements
   getLastPosition = unsupported
 }, InstantiationType.Delayed)
 
+registerSingleton(IAccessibleViewInformationService, class AccessibleViewInformationService implements IAccessibleViewInformationService {
+  _serviceBrand: undefined
+  hasShownAccessibleView = () => false
+}, InstantiationType.Delayed)
+
+registerSingleton(IAccessibilityService, class AccessibilityService implements IAccessibilityService {
+  _serviceBrand: undefined
+  onDidChangeScreenReaderOptimized = Event.None
+  onDidChangeReducedMotion = Event.None
+  alwaysUnderlineAccessKeys = async () => false
+  isScreenReaderOptimized = () => false
+  isMotionReduced = () => false
+  getAccessibilitySupport = () => 0
+  setAccessibilitySupport = unsupported
+  alert = unsupported
+  status = unsupported
+}, InstantiationType.Delayed)
+
```

Comment:
This service identifier is already registered with the real impl in the editor standalone services. It probably means the editor needs it, so registering it twice with a mocked impl is dangerous: if a race condition make this code execute before the standaloneServices registration, this empty impl will be used and it may break things

Reply:
Oh good one, removed it!